### PR TITLE
Support reading `locale` instead of `language`+`country` on templating

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

File: handlers/facebookapp/facebookapp.go [hunk 1]
```diff
@@ -1568,23 +1557,34 @@ func (h *handler) BuildAttachmentRequest(ctx context.Context, b courier.Backend,
...
+
+	// if we have a country, strip that off and look again for a match
+	l, c := lc.ToParts()
+	if c != "" {
+		if lang := supportedLanguages[courier.Locale(l)]; lang != "" {
+			return lang
+		}
+	}
+	return "en" // fallback to English
+}
```
Comment (rowanseymour) [inline handlers/facebookapp/facebookapp.go:1583]: there 2 changes to the logic here...

 1. If we're given a code like `swa-KE`, we try that, but also try `swa`
 2. If we don't find a match, fallback on `en`... this seems to me preferable to failing

@ericnewcomer @norkans7 thoughts?

File: handlers/facebookapp/facebookapp_test.go [hunk 2]
```diff
@@ -1526,3 +1534,13 @@ func TestBuildMediaRequest(t *testing.T) {
...
 }
+
+func TestGetSupportedLanguage(t *testing.T) {
+	assert.Equal(t, "en", getSupportedLanguage(courier.NilLocale))
+	assert.Equal(t, "en", getSupportedLanguage(courier.Locale("eng")))
+	assert.Equal(t, "en_US", getSupportedLanguage(courier.Locale("eng-US")))
+	assert.Equal(t, "pt_PT", getSupportedLanguage(courier.Locale("por")))
+	assert.Equal(t, "pt_PT", getSupportedLanguage(courier.Locale("por-PT")))
+	assert.Equal(t, "pt_BR", getSupportedLanguage(courier.Locale("por-BR")))
+	assert.Equal(t, "fil", getSupportedLanguage(courier.Locale("fil")))
```
Comment (norkans7) [inline handlers/facebookapp/facebookapp_test.go:1545]: we need an extra test for such `fr-CA` that should get `fr` too

Comment (norkans7) [reply handlers/facebookapp/facebookapp_test.go:1545]: ignoring the country

File: handlers/facebookapp/facebookapp.go (hunk 1 above)
Comment (norkans7) [reply handlers/facebookapp/facebookapp.go:1583]: That makes sense

**Review by norkans7 — APPROVED** [review]
Looks good, just the test for coverage

**Pull request merged**